packages/renderer/src/lib/kube/KubernetesDashboardGuideCard.spec.ts

```diff
@@ -0,0 +1,63 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import { render, screen } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+import { beforeAll, expect, test, vi } from 'vitest';
+
+import KubernetesDashboardGuideCard from './KubernetesDashboardGuideCard.svelte';
+
+const openExternalMock = vi.fn();
+
+// fake the window object
+beforeAll(() => {
+  (window as any).openExternal = openExternalMock;
```

Comment:
AFAIK the best way to not use any or unknown is
```suggestion
  Object.defineProperty(window, 'openExternal', {
    value: openExternalMock,
  });
```

Reply:
Fixed.